src/test/java/org/apache/commons/dbcp2/TestBasicDataSource.java

```diff
@@ -916,6 +917,34 @@ public void testCreateConnectionFactory() throws Exception {
         conn.close();
         ds.close();
     }
+
+    @Test
+    public void testEvict() throws Exception {
+        ds.setInitialSize(10);
+        ds.setMaxIdle(10);
+        ds.setMaxTotal(10);
+        ds.setMinIdle(5);
+        ds.setNumTestsPerEvictionRun(3);
+        ds.setMinEvictableIdleTimeMillis(100);
+        ds.setTimeBetweenEvictionRunsMillis(1000);
+        ds.setPoolPreparedStatements(true);
+
+        Connection conn = ds.getConnection();
+        conn.close();
+
+        ThreadMXBean threadBean = ManagementFactory.getThreadMXBean();
+        while (Arrays.stream(threadBean.getThreadInfo(threadBean.getAllThreadIds()))
+                .anyMatch(t -> t.getThreadName().equals("commons-pool-evictor-thread"))) {
+            if (ds.getNumIdle() <= ds.getMinIdle()) {
+                break;
+            }
+            Thread.sleep(1000);
```

Comment:
If this `1000` matches the one above in the call to `setTimeBetweenEvictionRunsMillis`, it would make sense to refactor the value in a local variable to make obvious that these must indeed match.

Reply:
Thanks for your comment.
The Evictor thread runs with a delay of the value of timeBetweenEvictionRunsMillis.
I think that the sleep time of the main thread should refer to that value.
I have fixed it.